src/coreclr/jit/targetwasm.cpp

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*****************************************************************************/
+
+#include "jitpch.h"
+#ifdef _MSC_VER
+#pragma hdrstop
+#endif
+
+#if defined(TARGET_WASM)
+
+#include "target.h"
+
+const char*            Target::g_tgtCPUName           = "wasm";
+const Target::ArgOrder Target::g_tgtArgOrder          = ARG_ORDER_R2L;
+const Target::ArgOrder Target::g_tgtUnmanagedArgOrder = ARG_ORDER_R2L;
+
+// clang-format off
+#ifdef UNIX_AMD64_ABI
+const regNumber intArgRegs [] = { REG_EDI, REG_ESI, REG_EDX, REG_ECX, REG_R8, REG_R9 };
```

Comment:
Is this still needed? I would expect this to needed once `codegen*` is excluded.